grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -112,15 +117,29 @@ public GraknEngineServer(GraknEngineConfig prop) {
         this.taskManager = startTaskManager(inMemoryQueue, redisCountStorage, jedisPool, lockProvider);
     }
 
-
     public static void main(String[] args) {
         GraknEngineConfig prop = GraknEngineConfig.create();
-        // Start Engine
+
+        // Start external components (Cassandra and Redis)
```

Comment:
With this approach how can I connect grakn to my own running instance of cass and redis? Can I even do that anymore? 

We don't formally support windows but I *think* this will kill off the "hacks" that people can do to get grakn running on windows. I.e. starting up cass and redis and configuring grakn to connect to them.